src/core/main.go

```diff
@@ -103,11 +105,32 @@ func main() {
 	redisURL := os.Getenv("_REDIS_URL")
 	if len(redisURL) > 0 {
 		gob.Register(models.User{})
-		beego.BConfig.WebConfig.Session.SessionProvider = "redis"
-		beego.BConfig.WebConfig.Session.SessionProviderConfig = redisURL
+		if strings.HasPrefix(redisURL, "redis+sentinel://") {
```

Comment:
This is great stuff!

But it seems the usage of `_REDIS_URL` is problematic, it has to support so many variations, any you need to introduce the `|` -> `,` magic...
If this is not a blocker, can we consider to provide a better way to set this configuration to `core`

For example, it may be the time we get rid of the env var `_RDIS_URL` but populate the information in to beego's configuration file during setup.

Reply:
If it is not too challenging, I also support @reasonerjt 's suggestion.

Reply:
这个变量只是针对 beego 的，而beego是写死的用了逗号和分号，所以只能找别的符号代替，管道符是我认为比较好的不会冲突的一个分割

Reply:
另外 redis+sentinel:// 这个写法是参考这个python和ruby的库
https://pypi.org/project/Redis-Sentinel-Url/
https://github.com/clearbit/redis-sentinel-url
其实也有其他的实现，如redis_sentinel

Reply:
That may be trick for helm-chart:
https://github.com/goharbor/harbor-helm/blob/76ad4f9c5c532280530ffbb9ce3c17e10a818ca1/templates/_helpers.tpl#L265

Reply:
原来的写法也是支持的，向后兼容，只是新增加了对sentinel的解析支持
而且helm之后肯定也是要改的，如果想支持sentinel的部署配置